pages/content-ui/src/index.tsx

```diff
@@ -1,8 +1,7 @@
 import { createRoot } from 'react-dom/client';
 import App from '@src/app';
-// eslint-disable-next-line
-// @ts-ignore
-import tailwindcssOutput from '@src/tailwind-output.css?inline';
+// @ts-expect-error That's because output file is create during build
+import tailwindcssOutput from '../dist/tailwind-output.css?inline';
```

Comment:
```suggestion
import tailwindcssOutput from '@src/tailwind-output.css?inline';
```

This is related to the comment above.